students/olszewski_bartosz/lesson_03_functions/negative_exponent.py

```diff
@@ -0,0 +1,13 @@
+def power(a, n):
+    result = 1
+    if n > 0:
+        for i in range(n):
+            result *= a
+    elif n < 0:
+        a = 1 / a
+        for i in range(abs(n)):
+            result *= a
```

Comment:
Try to avoid duplication of code

```
def power(a, n):
    result = 1
    for i in range(abs(n)):
        result *= a
    if n >= 0:
        return result
    else:
        return 1 / result
```